.github/workflows/macos-ci.yml

```diff
@@ -21,8 +21,8 @@ jobs:
       fail-fast: false
       matrix:
         os:
-          #- macos-11
-          - macos-12
+          - macos-13
+          # - macos-14
```

Comment:
On macOS 14, the CI got as far as 50% through compiling the code, but then erred out at that point. I decided to leave macOS 14 for later.